index.bs

```diff
@@ -488,13 +490,238 @@ partial interface NeuralNetworkContext {
     </div>
 </div>
 
+### gru ### {#api-neuralnetworkcontext-gru}
+Gated Recurrent Unit (GRU) recurrent network using an update gate and a reset gate to compute the hidden state that rolls into the output across the temporal sequence of the network, as outlined in this [paper](https://arxiv.org/pdf/1406.1078.pdf).
```

Comment:
I suggest you make this link a biblio reference (add into `<pre class="biblio">`) and point to it instead. This allows the reader to see all the references at https://webmachinelearning.github.io/webnn/#references and makes a clear distinction between normative (`[[!foo]]`) and informative references (`[[foo]]`). 

(I notice this "in this paper" convention is used in batchNormalization prose as well, so that could be aligned similarly as well.)